doc/problem/ultradian.md

```diff
@@ -18,15 +12,60 @@ Ultradian rhythm is widely observed in mammalian behavioural patterns. Ultradian
     - Ultradian rhythms could be treated as one feature for clustering the 16 strains. We may also subset the data using the results of the cluster and analysis the rhythm similarities and differences across clusters. 
 
+## Statement of statistical problem
+- How to determine the optimal bin intervals for constructing the time series?
+    - The bin interval may vary according to the frequency. Bin interval examples: 5 min, 30 min, 1 hour etc. Need to look into the data. 
+- How to test the autocorrelation coefficient for the data and assess the model?
+    -  Use the AIC to select best time lags for the time series model and the K statistics to test the goodness of fit.
+- For longitudinal data analysis, how to build the model? Which is the fixed effect or  random effect?
+
+
+## Exploratory Analysis
+- Data investigation
+    - Think about known/expected cycles - time to digest, IS/AS cycle, etc.
+    - Try to investigate cycles that are greater than 24 hours to avoid missing cycles.
+    - During the acclimatization period, investigate difference in cycles.
+- Plots
+    - Plots for determining optimal bin intervals for constructing the time series.
+    - Plots for discovering the frequency or period.
+    - General time series plots for getting intuitions for each variables.
+- Models
+    - Usage of Lomb-Scargle (LS) periodogram spectral analysis technique, a widely used tool in period detection and frequency analysis
+
+## Data Requirements Description
+- Input:  records for each strains (total of  16), each feature of interest (food, water, distance, active_state probability ...), in a duration of 12 days (excluding 4 acclimation days) . 
+- Processed: using one-minute time bins of movement records to binary score the activity into 0 (IS: inactive state) and 1 (AS: active state); using thirty-minute bins of food records to calculate the amount of chows consumed by mice; using LS periodogram technique to select the appropriate time bins for above.
+- Output: different patterned visualization for each feature, with the appropriate time bins that presents the most significant ultradian pattern.
+
+
+
 ## Method and models
 - Seasonal decomposition.
+    - Decomposition based on the rate of change.
+        - the Trend Component  that reflects the long term progression of the series (secular variation)
+        - the Cyclical Component  that describes repeated but non-periodic fluctuations
+        - the Seasonal Component  reflecting seasonality (seasonal variation)
+        - the Irregular Component  (or "noise") that describes random, irregular influences. It represents the residuals of the time series after the other components have been removed.
+
+    - Decomposition based on predictability.
 - Longitudinal data analysis.
-- Autocorrelation anslysis.
+    - Definition: A longitudinal study refers to an investigation where participant outcomes and possibly treatments or exposures are collected at multiple follow-up times.
```

Comment:
You may either remove this or cite the source where this definition is from. Based on a google search, this seems to come from the lecture notes for a course in University of Washington.